src/main/java/org/jabref/model/pdf/search/PdfSearchResults.java

```diff
@@ -0,0 +1,45 @@
+package org.jabref.model.pdf.search;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class PdfSearchResults {
+
+    private final List<SearchResult> searchResults;
+
+    public PdfSearchResults(List<SearchResult> search) {
+        this.searchResults = Collections.unmodifiableList(search);
+    }
+
+    public PdfSearchResults() {
+        this.searchResults = Collections.unmodifiableList(Collections.emptyList());
+    }
+
+    public List<SearchResult> getSortedByScore() {
+        List<SearchResult> sortedList = new ArrayList<>(searchResults);
+        sortedList.sort((searchResult, t1) -> {
+            if (searchResult.getLuceneScore() < t1.getLuceneScore()) {
+                return -1;
+            }
+            if (searchResult.getLuceneScore() > t1.getLuceneScore()) {
+                return 1;
+            }
+            return 0;
+        });
+        return Collections.unmodifiableList(sortedList);
+    }
+
+    private List<SearchResult> getSortedByAlphabet() {
+        // TODO implement sorting
+        throw new RuntimeException("Not implemented");
+    }
```

Comment:
Whats the plan here?

Reply:
Leftover from @LinusDietz. I guess it was just an idea, I don't see a usecase for it right now and would vote to remove it.

Reply:
(talking about getSortedByAlphabet only, getSortedByScore is already used to put the most prominent search-results on top in the search results tab)